published copy: home paths removed, 0 em dashes or middle dots in context lines replaced by a hyphen
diff --git i/lib/features/home/presentation/screens/home_screen.dart w/lib/features/home/presentation/screens/home_screen.dart
index c896794e..16fddc19 100644
--- i/lib/features/home/presentation/screens/home_screen.dart
+++ w/lib/features/home/presentation/screens/home_screen.dart
@@ -919,7 +919,7 @@ class _HomeScreenState extends ConsumerState<HomeScreen> {
               right: mapChromeMargin,
               bottom: bottomNavClearance(context) + _pinCardGap,
               child: SizeReporter(
-                onSize: _onDotCardSize,
+                onSize: (_) {},
                 child: CompositedTransformTarget(
                   link: _dotCardLink,
                   child: MapPointCard(
